docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+![m'lady](https://cdn-images-1.medium.com/max/367/1*6j17ZDuywkKu7TOa2yvAKg.png)
+
+
+# A Step-by-Step Guide on Streaming Data from AWS RDS Multi-AZ PostgreSQL DB to a Standlone PostgreSQL DB Using Debezium 0.7.1 Connector :
+
+In this guide, I will share my experience on integrating AWS RDS PostgreSQL and Debezium Kafka 
+Connector. So we will be able to stream data from AWS RDS PostgreSQL to an another standlone 
+PostgreSQL DB on AWS EC2 Server.
+
+Before working with Debezium, you need to make a decision on which connector will be "Source Connector" and 
+which one will be "Sink Connector". In our example, "Source Connector" will connect to "AWS RDS Multi-AZ PostgreSQL DB, 
+and "Sink Connector" will connect to Standlone PostgreSQL DB. 
+
+Actually, the configuration below supports two-way connectors as well. Please keep in mind that only a single 
+Debezium Kafka Connector can serve multiple tasks, it means to be able to use the same connector as source and 
+sink  and can work one-to-many, many-to-one, one-to-one ways. 
+
+Note : You can download the necessary debezium AWS RDS config scripts from https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts
+
+---
+### Prerequisites:
+To be able to understand all the concepts mentioned here properly, you should be familiar with the systems 
+like; debezium, docker, docker-compose, AWS RDS, PostgreSQL, Apache Kafka, Linux.
+The guide assumes that you have setup an instance on AWS and installed the necessary apps on it. 
+Before we go;
+1. First, you need to create an AWS EC2 Instance with Amazon Linux AMI as explained at this link. 
+`https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launching-instance.html`
+
+2. Second, it is required to install docker on the machine as explained at this link. 
+`https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html`
+
+3. Lastly, it is required to install docker compose as explained at this link.
+`https://docs.docker.com/compose/install/#install-compose`
+---
+
+<br>
+<br>
+
+## 1. Streaming Data From AWS RDS PostgreSQL DB to PostgreSQL DB:
+
+---
+### 1.1 Topology
+
+![m'lady](./images/debezium-postgres.jpg)
+<br>
+---
+
+<br>
+<br>
+
+### 1.2 AWS RDS PostgreSQL DB Configuration
+
+1. Create a new "Parameter Group" at AWS RDS ***Parameter groups*** section and give it a name , for example, "postgres-db-parameter-groups"
+
+2. Be sure to set the new ***Parameter Group*** parameters as follows:
+    
+![m'lady](./images/AWS-RDS-Postgres-DB-Parameter-Group.png) 
+	
+	
+----
+	rds.logical_replication = 1
+    shared_preload_libraries = pg_stat_statements,pg_hint_plan,pgaudit
+    ssl=0
+----
+
+* 3 - Create a new AWS RDS PostgreSQL DB called testdbserver. Set the user name as ***testdbuser***. 
+Give permission to Port Number ***TCP:5432*** to all sources (0.0.0.0/0) via ***DB Security Group***.
+
+![m'lady](./images/AWS-Security-Group-permission.png) 
+
+4. Connect to the database via psql 
```

Comment:
The numbering starts again here in the rendered output. I recommend you build the website as described [here](https://github.com/debezium/debezium.github.io) to make sure the output is correctly formatted.